nilearn/plotting/html_surface.py

```diff
@@ -0,0 +1,584 @@
+    anat_color = mpl.cm.get_cmap('Greys')(anat_norm(sulc_depth_map))
+    vertexcolor[np.abs(surf_map) < absolute_threshold] = anat_color[
+        np.abs(surf_map) < absolute_threshold]
+    return _to_color_strings(vertexcolor)
+
+
+def one_mesh_info(surf_map, surf_mesh, threshold=None, cmap=cm.cold_hot,
+                  black_bg=False, bg_map=None, symmetric_cmap=True):
+    info = {}
+    colors, cmin, cmax, cmap, norm, abs_threshold = colorscale(
+        cmap, surf_map, threshold, symmetric_cmap=symmetric_cmap)
+    info['inflated_left'] = to_plotly(surf_mesh)
+    info['vertexcolor_left'] = _get_vertexcolor(
+        surf_map, cmap, norm, abs_threshold, bg_map)
+    cmin, cmax = float(cmin), float(cmax)
+    info["cmin"], info["cmax"] = cmin, cmax
+    info['black_bg'] = black_bg
+    info['full_brain_mesh'] = False
+    return info, colors
+
+
+def _check_mesh(mesh):
+    if not isinstance(mesh, str):
+        assert isinstance(mesh, collections.Mapping)
+        assert {'pial_left', 'pial_right', 'sulc_left', 'sulc_right',
+                'infl_left', 'infl_right'}.issubset(mesh.keys())
+        return mesh
+    if mesh != 'fsaverage5':
+        raise ValueError(
+            'mesh should be fsaverage5 until fsaverage fetcher pr is merged')
+    mesh = datasets.fetch_surf_fsaverage5()
+    return mesh
+
+
+def full_brain_info(stat_map, mesh='fsaverage5', threshold=None,
+                    cmap=cm.cold_hot, black_bg=False, symmetric_cmap=True,
+                    vol_to_surf_kwargs={}):
+    info = {}
+    mesh = _check_mesh(mesh)
+    surface_maps = {
+        h: surface.vol_to_surf(stat_map, mesh['pial_{}'.format(h)],
+                               **vol_to_surf_kwargs)
+        for h in ['left', 'right']
+    }
+    colors, cmin, cmax, cmap, norm, abs_threshold = colorscale(
+        cmap, np.asarray(list(surface_maps.values())).ravel(), threshold,
+        symmetric_cmap=symmetric_cmap)
+
+    for hemi, surf_map in surface_maps.items():
+        sulc_depth_map = surface.load_surf_data(mesh['sulc_{}'.format(hemi)])
+        info['pial_{}'.format(hemi)] = to_plotly(mesh['pial_{}'.format(hemi)])
+        info['inflated_{}'.format(hemi)] = to_plotly(
+            mesh['infl_{}'.format(hemi)])
+
+        info['vertexcolor_{}'.format(hemi)] = _get_vertexcolor(
+            surf_map, cmap, norm, abs_threshold, sulc_depth_map)
+    cmin, cmax = float(cmin), float(cmax)
+    info["cmin"], info["cmax"] = cmin, cmax
+    info['black_bg'] = black_bg
+    info['full_brain_mesh'] = True
+    return info, colors
+
+
+def _fill_html_template(info, colors, embed_js=True):
+    as_json = json.dumps(info)
+    as_html = HTML_TEMPLATE.replace('INSERT_STAT_MAP_JSON_HERE', as_json)
+    as_html = as_html.replace('INSERT_COLORSCALE_HERE', colors)
+    as_html = add_js_lib(as_html, embed_js=embed_js)
+    return HTMLDocument(as_html)
+
+
+def view_img_on_surf(stat_map, mesh='fsaverage5',
```

Comment:
We have the convention that niimg-like variables are named with a name ending by "_img".